### PR TITLE
refactor(app): Display pipette offsets in calibration card

### LINKED ISSUE #6612: Cal Interconnections: Subpage: List Pipettes
Add a block to the calibration subpage from #6611 for pipette offset calibrations.

The link should navigate to the pipettes & offset pages

Calibration status for left and right pipette should be shown, including datestamp of calibration (if any) and what tiprack the calibration was executed with.

Implement as per [the designs](https://www.figma.com/file/nbDHskbW5zaXwtrLvEIKVj/calibration-overhaul?node-id=124%3A410); see the various pipette scenarios for how to handle different combinations of attached and calibrated pipettes.

### PR DESCRIPTION
**Pull request by sfoster1**
Add information displaying the pipette offset calibration for currently attached pipettes (per [the design](https://www.figma.com/file/nbDHskbW5zaXwtrLvEIKVj/calibration-overhaul?node-id=124%3A0)) to the calibration card.

This has a couple other things that were required:
- Expose the `tiprackUri` component of the pipette offset calibration via HTTP; it was previously saved but not exposed
- Fetch pipettes and pipette offset calibrations when the calibration page is displayed
- New selector to fuse attached pipettes and offset calibrations

## Risk
None, behind feature flag

## Review Requests

- Is this the right way / place to make the new selector?
- Is this the right way / place to fetch data?
- General code review

## Testing
- Make sure you put the robot server build from this PR on your robot and turn on the calibration overhaul flags on both app  and robot
- stop the API server and restart the app.
   - Look at the robot page for the unhealthy robot and see that there are no pipettes connected
   - Restart the api server while continuing to look at the robot page; when the api server comes up, the robot page should refresh with attached pipettes and calibration on its own
- check that the proper data for
  - attached and calibrated
  - not attached 
  - attached and not calibrated
   work
- check that the right tiprack is displayed. In theory this should work with a custom tiprack.

Closes #6612

### CODE REVIEW

**Comment by Laura-Danielle** [comment]
Not sure if we decided not to follow the design exactly, but for the case when there is no pipette attached there should be two N/As. One for the pip name and one for the calibration section.

Screen Shot 2020-10-02 at 9 19 32 AM

**Review by emilywools — APPROVED** [review]

File: app/src/components/RobotSettings/PipetteOffsetItem.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+export function PipetteOffsetItem(props: Props): React.Node {
+  const { mount, pipette, calibration, customLabware } = props
+  return (
+    <Box width={'50%'} display={DISPLAY_INLINE_BLOCK}>
+      <Text
+        as={'h4'}
+        textTransform={TEXT_TRANSFORM_UPPERCASE}
+        fontWeight={FONT_WEIGHT_BOLD}
```
Comment (b-cooper) [inline app/src/components/RobotSettings/PipetteOffsetItem.js:92]: This font weight (800) is heavier than the designs.  The bold text in the designs is 700.  That said, I don't think we use 800 anywhere else across the app. Maybe FONT_WEIGHT_BOLD should actually be 700?

**Comment by sfoster1** [comment]
Screenshot from 2020-10-05 13-12-17
Changed to 600 after discussion

**Review by emilywools — APPROVED** [review]
600 weight looks better than 800 weight. thank you for calling it out and updating!

File: app/src/components/RobotSettings/PipetteOffsetItem.js (hunk 1 above)
Comment (sfoster1) [reply app/src/components/RobotSettings/PipetteOffsetItem.js]: changed to `SEMIBOLD`. 800 does look weird for bold but semibold is better here I think

File: app/src/components/RobotSettings/PipetteOffsetItem.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    </>
+  ) : (
+    <Text fontStyle={FONT_STYLE_ITALIC}>{NO_CALIBRATION}</Text>
+  )
+}
+
+export function PipetteOffsetItem(props: Props): React.Node {
+  const { mount, pipette, calibration, customLabware } = props
+  return (
+    <Box width={'50%'} display={DISPLAY_INLINE_BLOCK}>
```
Comment (b-cooper) [inline app/src/components/RobotSettings/PipetteOffsetItem.js:88]: This largely comes down to preference, but just so you know, this layout is just as achievable using flexbox.  e.g. 
```
<Flex flex="1" flexDirection={DIRECTION_COLUMN}>
```

**Review by b-cooper — APPROVED** [review]
Code looks good, and things are working as expected from what I can tell 🥼

**Pull request merged**